### PR TITLE
Add Repose v8 support [Rep 4032]

### PR DESCRIPTION
**Pull request by JoelRizner**
Upgraded the filter to be compatible with Repose 8. Functionally the custom strings for the response message has been removed.

### CODE REVIEW

File: extract-device-id/src/main/java/org/openrepose/filters/custom/extractdeviceid/ExtractDeviceIdFilter.java [hunk 1]
```diff
@@ -123,18 +125,19 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
             LOG.error("Extract Device ID filter has not yet initialized...");
             ((HttpServletResponse) servletResponse).sendError(SC_INTERNAL_SERVER_ERROR); // (500)
         } else {
-            MutableHttpServletRequest mutableHttpRequest = MutableHttpServletRequest.wrap((HttpServletRequest) servletRequest);
-            HttpServletResponse httpServletResponse = ((HttpServletResponse) servletResponse);
+            HttpServletRequestWrapper wrappedHttpRequest = new HttpServletRequestWrapper((HttpServletRequest) servletRequest);
+            HttpServletResponseWrapper wrappedHttpResponse = new HttpServletResponseWrapper(
```
Comment (adrianjgeorge) [inline extract-device-id/src/main/java/org/openrepose/filters/custom/extractdeviceid/ExtractDeviceIdFilter.java]: why did you wrap this?

Comment (JoelRizner) [inline extract-device-id/src/main/java/org/openrepose/filters/custom/extractdeviceid/ExtractDeviceIdFilter.java]: The first thing Mario had me change was this part of the filter, to match these repose-hello-world changes: https://github.com/rackerlabs/repose-hello-world/commit/58618b4f4b2cc07392b6106de8452fda0a8a5b96

This was one of the changes.

Comment (JoelRizner) [inline extract-device-id/src/main/java/org/openrepose/filters/custom/extractdeviceid/ExtractDeviceIdFilter.java]: I see now that the response was not wrapped before so i will unwrap the response.

**Comment by mburns** [comment]
looks reasonable to me.

**Comment by mburns** [comment]
@JoelRizner is this version supported in Repose 8-only, or both version 7 and 8?

**Comment by Mario-Lopez** [comment]
@mburns This version supports only Repose 8.

**Comment by dimtruck** [comment]
@mburns @jjbuchan we should merge this...no?

**Comment by mburns** [comment]
@dimtruck yes, yes, we should.

**Comment by jjbuchan** [comment]
Oh, look at this.  Could be helpful

**Pull request merged**